src/index.js

```diff
@@ -70,23 +71,30 @@ const elmWebComponents = {
           const flags = mapFlags(props)
           context.flags = flags
 
+          var elmDiv = this;
+          var parentDiv = this;
```

Comment:
We should use `const`/`let` rather than `var` here.

```
const parentDiv = useShadowDom ? this.attachShadow({ mode: 'open' }) : this;
```

maybe?

Reply:
Indeed that's cleaner. And combined with your previous comment, I can use `const elmDiv in the 0.19 branch.